emanate/__init__.py

```diff
@@ -15,14 +15,14 @@
 import sys
 from .config import Config
 
-# Expose `emanate.version.__version__` as `emanate.__version__`.
-from .version import __version__  # noqa: F401
-
-
-#: Emanate's maintainer.
-__author__ = "Ellen Marie Dash"
-
-# __version__ is defined in version.py.
+try:
+    from importlib import metadata
+except ImportError:
+    import importlib_metadata as metadata  # type: ignore
+
+# Set the module-level dunders suggested in PEP8
+__author__ = metadata.metadata('emanate').get('author')
```

Comment:
Is there a way to pass `metadata` the current module, instead of hardcoding a name?

Reply:
`__name__`